blueprint.json

```
@@ -0,0 +1,18 @@
+{
+	"$schema": "https://playground.wordpress.net/blueprint-schema.json",
+	"landingPage": "\/wp-admin\/admin.php?page=bp-components",
+	"preferredVersions": {
+		"php": "8.1",
+		"wp": "6.5"
```

Is it possible to set it to `latest`?

---

It is! Just checked: https://wordpress.github.io/wordpress-playground/blueprints-api/index/
```suggestion
		"wp": "latest"
```